_data/components/omaplayer.yml

```diff
@@ -0,0 +1,97 @@
+directive : "o-map-layer"
+
+attributes: [{
+  name: "layer-center",
+  description: "Center the position of the layer",
```

Comment:
Explain that the structure is latitude,longitude separated by comma and maybe give an example